db/compaction/compaction_picker_test.cc

```diff
@@ -3178,11 +3178,7 @@ TEST_F(CompactionPickerTest, UniversalMarkedL0Overlap2) {
       universal_compaction_picker.PickCompaction(
           cf_name_, mutable_cf_options_, mutable_db_options_, vstorage_.get(),
           &log_buffer_));
-  ASSERT_TRUE(compaction2);
-  ASSERT_EQ(3U, compaction->num_input_files(0));
-  ASSERT_TRUE(file_map_[1].first->being_compacted);
-  ASSERT_TRUE(file_map_[2].first->being_compacted);
-  ASSERT_TRUE(file_map_[4].first->being_compacted);
+  ASSERT_FALSE(compaction2);
```

Comment:
Didn't figure out why things changed here.

Reply:
reverted the change, for L0 output, they can actually overlapped.